include/wolfssl/meson.build

```diff
@@ -0,0 +1,8 @@
+if 'aarch64'.contains(host_machine.cpu_family())
+
+    wolfssl_options = configure_file(input: 'options_aarch64.in', output: 'options.h', configuration: cdata)
```

Comment:
Any chance we can support 32bit ARM? Running netatalk on f.e. a RPi Zero is a fairly common usecase.

Reply:
Working on it...

Reply:
Done